.github/workflows/check_pr

```diff
@@ -0,0 +1,25 @@
+name: Check Pull Request
```

Comment:
do you have a link to where you found it out?

Reply:
took it s bit from work repos  but the gist of it is here: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-the-python-starter-workflow